### PR TITLE
Updated Example

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Hi there,

For my last PR I build the example with the newest react-native version, so I simply wanted to share it.
I am not sure how much focus this project has on backwards compatibility. If we want to support this even better, we should decline this PR and move an example into a folder with the used RN version number, eg:

```
├── 0.13
│   └── DrawerLayoutExample
├── 0.14
│   └── DrawerLayoutExample
├── 0.15
│   └── DrawerLayoutExample
├── 0.16
│   └── DrawerLayoutExample
└── 0.17
    └── DrawerLayoutExample
```

This way we would also be able to track down bugs even faster.

Please note, that I didn't use the actual package in the `package.json` but a file reference, that way it is easier for testing, but I am not sure if it may be confusing to potential users. Should I add a note to this line?

Happy Holidays!

### CODE REVIEW

File: DrawerLayoutExample/package.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
...
   "private": true,
   "scripts": {
-    "start": "node_modules/react-native/packager/packager.sh"
+    "start": "react-native start"
   },
   "dependencies": {
-    "react-native": "^0.11.0",
-    "react-native-drawer-layout": "^0.1.0"
+    "react-native": "^0.17.0",
+    "react-native-drawer-layout": "file:.."
```
Comment (mfmendiola) [inline DrawerLayoutExample/package.json:10]: Hm, do you want to keep this referring to the relative path?

**Comment by mfmendiola** [comment]
lgtm. I don't think we need to go to the extent of having a separate example for each of the RN versions, unless there's a breaking change.  We should, however, be transparent in the documentation about the versions that we currently support.

File: DrawerLayoutExample/package.json (hunk 1 above)
Comment (DanielMSchmidt) [inline DrawerLayoutExample/package.json:10]: Yes, as it makes it easier to test changes in the lib in this example application. (only dist/ and the package.json is copied, so the behavior should be the same)
Are there drawbacks I don't see or is it just unusual to do?

**Comment by DanielMSchmidt** [comment]
@mfmendiola Do you know any versions we currently don't support?

**Comment by mfmendiola** [comment]
As far as I know, we should be supporting >= v0.11.0, which was the release with Android support https://github.com/facebook/react-native/releases/tag/v0.11.0.

**Comment by DanielMSchmidt** [comment]
Great, I added a line in the Readme about that. Wanted to move to RN 0.18 which was just released, but somehow npm doesn't know about the release yet. If you are ok with the changes I would merge this PR and open a (hopefully) small one with RN 0.18 later on ;)

File: DrawerLayoutExample/package.json (hunk 1 above)
Comment (mfmendiola) [inline DrawerLayoutExample/package.json:10]: No, that's fine. Just making sure it wasn't residual from debugging :smile:

**Comment by mfmendiola** [comment]
lgtm..feel free to merge when ready

**Pull request merged**